server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -130,19 +130,15 @@ public void unregister(CommandHandler handler) {
     /**
      * Verifies if the command can be posted to this {@code CommandBus}.
      *
-     * <p>The command can be posted if it has either dispatcher or handler registered with
+     * <p>A command can be posted if it has either dispatcher or handler registered with
      * the command bus.
```

Comment:
Please change `the command bus` to `this {@code CommandBus}. It would match the syntax of a similar method in `EventBus`.